packages/swagger-editor/src/components/SwaggerEditor/HeaderTabButton.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+import { useMediaQuery } from '@vueuse/core'
+
+defineProps<{
+  title: string
```

Comment:
I usually prefer to make content like this a slot then you can turn parts of the text on or off in the parent

(Kind of a composition vs configuration thing)

Reply:
e.g.

```
<HeaderTabButton @click="handleMe">
  <span v-if="isMobile">Import&nbsp;</span>URL
</HeaderTabButton>
```